apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx

```diff
@@ -1,24 +1,25 @@
 import { MegaphoneIcon } from "@heroicons/react/24/outline";
+import { CalendarButton } from "@namehash/internal";
 import { Link } from "@namehash/namekit-react";
 
 export const HeadlineBanner = () => {
```

Comment:
Please fix the height of the banner. It's too big.

![CleanShot 2024-11-09 at 21 13 42](https://github.com/user-attachments/assets/3655ffc8-3a5c-4d01-ad10-5693092ee2b5)